### ISSUE TITLE
ERROR: no certificate subject alternative name matches requested host name nightlies.plex.tv

### ISSUE THREAD

**Issue by dewey** [foux/fouxlay#13]
Trying to install the latest version of `plex-media-server` causes this error, any idea what's wrong there?

```
Calculating dependencies... done!
[ebuild     U ~] media-tv/plex-media-server-0.9.12.3::fouxlay [0.9.11.17::fouxlay] 124,769 KiB

Total: 1 package (1 upgrade), Size of downloads: 124,769 KiB

Would you like to merge these packages? [Yes/No] yes

>>> Verifying ebuild manifests

>>> Emerging (1 of 1) media-tv/plex-media-server-0.9.12.3::fouxlay
>>> Downloading 'http://distfiles.gentoo.org/distfiles/plexmediaserver_0.9.12.3.1173-937aac3_amd64.deb'
--2015-06-05 13:56:07--  http://distfiles.gentoo.org/distfiles/plexmediaserver_0.9.12.3.1173-937aac3_amd64.deb
Resolving distfiles.gentoo.org... 216.165.129.135, 137.226.34.42, 140.211.166.134, ...
Connecting to distfiles.gentoo.org|216.165.129.135|:80... connected.
HTTP request sent, awaiting response... 404 Not Found
2015-06-05 13:56:07 ERROR 404: Not Found.

>>> Downloading 'http://nightlies.plexapp.com/directdl/plex-media-server/dist-ninja/0.9.12.3.1173-937aac3/plexmediaserver_0.9.12.3.1173-937aac3_amd64.deb'
--2015-06-05 13:56:07--  http://nightlies.plexapp.com/directdl/plex-media-server/dist-ninja/0.9.12.3.1173-937aac3/plexmediaserver_0.9.12.3.1173-937aac3_amd64.deb
Resolving nightlies.plexapp.com... 2400:cb00:2048:1::6814:d1d, 2400:cb00:2048:1::6814:c1d, 104.20.12.29, ...
Connecting to nightlies.plexapp.com|2400:cb00:2048:1::6814:d1d|:80... connected.
HTTP request sent, awaiting response... 302 Moved Temporarily
Location: https://nightlies.plex.tv./directdl/plex-media-server/dist-ninja/0.9.12.3.1173-937aac3/plexmediaserver_0.9.12.3.1173-937aac3_amd64.deb [following]
--2015-06-05 13:56:08--  https://nightlies.plex.tv./directdl/plex-media-server/dist-ninja/0.9.12.3.1173-937aac3/plexmediaserver_0.9.12.3.1173-937aac3_amd64.deb
Resolving nightlies.plex.tv.... 2400:cb00:2048:1::6814:709, 2400:cb00:2048:1::6814:609, 104.20.6.9, ...
Connecting to nightlies.plex.tv.|2400:cb00:2048:1::6814:709|:443... connected.
ERROR: no certificate subject alternative name matches
	requested host name ‘nightlies.plex.tv.’.
To connect to nightlies.plex.tv. insecurely, use `--no-check-certificate'.
!!! Couldn't download 'plexmediaserver_0.9.12.3.1173-937aac3_amd64.deb'. Aborting.
 * Fetch failed for 'media-tv/plex-media-server-0.9.12.3', Log file:
 *  '/var/tmp/portage/media-tv/plex-media-server-0.9.12.3/temp/build.log'

>>> Failed to emerge media-tv/plex-media-server-0.9.12.3, Log file:

>>>  '/var/tmp/portage/media-tv/plex-media-server-0.9.12.3/temp/build.log'

 * Messages for package media-tv/plex-media-server-0.9.12.3:

 * Fetch failed for 'media-tv/plex-media-server-0.9.12.3', Log file:
 *  '/var/tmp/portage/media-tv/plex-media-server-0.9.12.3/temp/build.log'
```

**Comment by MJDSys** [comment]
I just ran into this myself, wget for some reason refuses to work with a redirection as occurs in the above case.  Changing the SRC_URI to point to nightlies.plex.tv instead of nightlies.plexapp.com resolves it for now.

Since this seems to be a wget bug, @dewey could you post the output of ```emerge -vp --nodeps wget```?

@foux Could you change the SRC_URI?  Would you prefer me to whip up a PR for you?

**Comment by dewey** [comment]
@MJDSys That's the output:

```
root@host ~$ emerge -vp --nodeps wget                                                                      1 ↵

These are the packages that would be merged, in order:

[ebuild   R    ] net-misc/wget-1.16::gentoo  USE="ipv6 nls pcre ssl zlib -debug -gnutls -idn -ntlm -static {-test%} -uuid" 0 KiB

Total: 1 package (1 reinstall), Size of downloads: 0 KiB
```

**Comment by MJDSys** [comment]
@dewey Thanks, it seems we have quite different use flags, so this will be interesting to track down.

**Comment by foux** [comment]
Sorry for the lack of response. It's fixed with the version 0.9.12.4

**Comment by dewey** [comment]
@foux It looks like this just happened again with the latest update #17 - Should be re-opened.
